features/file-system-access.yml

```
@@ -0,0 +1,12 @@
+name: File system access
+description: The `showOpenFilePicker()`, `showDirectoryPicker()`, and `showSaveFilePicker()` functions, and the `FileSystemHandle` interface access files and directories on the user's device and allow you to read and write to them.
```

```suggestion
description: The `showOpenFilePicker()`, `showDirectoryPicker()`, and `showSaveFilePicker()` functions, and the `FileSystemHandle` interface allow for getting access to files and directories on the user's device and read and (if the user allows) write to them.
```

Not sure if it's worth mentioning that `FileSystemHandle` and its children are defined in the actual WHATWG FS standard, whereas the picker methods are still only in WICG. They share the `FileSystemHandle`.

---

I agree that we don't need to mention the handle interface, especially since it doesn't look as if developers will frequently call that API by name. It looks likely they'll use the objects from the `show` methods instead.

For brevity then, suggesting:

```suggestion
description: The `showOpenFilePicker()`, `showDirectoryPicker()`, and `showSaveFilePicker()` methods request access to files and directories on the user's device and returns a handle for reading and writing to them.
```